ignite/contrib/metrics/precision_recall_curve.py

```diff
@@ -73,3 +75,32 @@ def __init__(self, output_transform: Callable = lambda x: x, check_compute_fn: b
         super(PrecisionRecallCurve, self).__init__(
             precision_recall_curve_compute_fn, output_transform=output_transform, check_compute_fn=check_compute_fn
         )
+
+    def compute(self) -> float:
```

Comment:
3 values are returned. You must fix the function signature.

Reply:
sorry, overlooked it.